### DISCUSSION TITLE
Why i can´t delete button

### DISCUSSION THREAD

**Discussion by CeContreras** [TomSchimansky/CustomTkinter#1730] [Q&A]
![image](https://github.com/TomSchimansky/CustomTkinter/assets/42829956/2e695f3a-6e3f-47a7-9801-8b4fc6e9b60b)
I want delete the button create with customtkinter, however i am used the funtion .destroy() but not delete

**Comment by ghost** [comment]
Could you remove the `try, except` statement and then give us the full traceback? That might provide some more information.

**Comment by avalon60** [comment]
Difficult to judge with a small snippet of code, but have you tried removing the try/except block to see if an exception is being raised?
